scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/tssop.yaml

```diff
@@ -517,6 +517,27 @@ TSSOP-24_4.4x7.8mm_P0.65mm:
   num_pins_x: 0
   num_pins_y: 12
 
+TSSOP-24-1EP_4.4x7.8mm_P0.65mm_EP3.2x5mm:
+  size_source: 'JEDEC MO-153 Var AD https://www.jedec.org/document_search?search_api_views_fulltext=MO-153'
+  body_size_x: 4.4 +/-0.1
+  body_size_y: 7.8 +/-0.1
+  overall_size_x: 6.4
+  body_height: 1.2
+  lead_width: 0.17 .. 0.30
+  lead_len: 0.6 +/-0.15
+  pitch: 0.65
+  num_pins_x: 0
+  num_pins_y: 12
+  EP_size_x:
+    minimum: 3
+    nominal: 3.2
+    maximum: 3.4
+  EP_size_y:
+    minimum: 4.8
+    nominal: 5
+    maximum: 5.2
+  EP_num_paste_pads: [1, 1]
```

Comment:
This being a relatively large EP, please split into more paste pads (like [2, 3] or possibly [3, 4])

Reply:
Hi @chschlue Thank you for the review, Just commited changes.
Thanks